### ISSUE TITLE
Upgrade github.com/go-openapi/strfmt to latest version to patch mongo-driver vulnerability alert

### ISSUE THREAD

**Issue by arvchristos** [lightningnetwork/lnd#6293]
### Background

We recently got a Dependabot alert regarding `go.mongodb.org/mongo-driver`. Tracing the dependency graph, this dependency comes from `lnd` and especially `lnd -> github.com/go-openapi/strfmt@v0.19.5 -> go.mongodb.org/mongo-driver`.

lnd fetches a version of `strfmt` that is using a vulnerable `mongo-driver` version. Developers of `strfmt` have released a new version to tackle this. Could you please update the `strfmt` version used?

**Comment by Roasbeef** [comment]
Related issue: https://github.com/lightningnetwork/lnd/issues/5740

**Comment by Roasbeef** [comment]
> Tracing the dependency graph, this dependency comes from lnd and especially
```
⛰   go mod why github.com/go-openapi/strfmt
go: downloading github.com/onsi/ginkgo v1.7.0
go: downloading github.com/onsi/gomega v1.4.3
go: downloading github.com/tidwall/pretty v1.0.0
go: downloading github.com/ulikunitz/xz v0.5.8
go: downloading golang.org/x/tools v0.1.8
go: downloading github.com/hpcloud/tail v1.0.0
go: downloading gopkg.in/fsnotify.v1 v1.4.7
# github.com/go-openapi/strfmt
github.com/lightningnetwork/lnd/cmd/lncli
github.com/jedib0t/go-pretty/text
github.com/go-openapi/strfmt
```

So it's actually an indirect dependnacy, fixed in #6295.
